SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -113,63 +128,68 @@ etc...
 
     > These lines ensure that the SpatialOS Unreal GDK can override Unreal's network replication.
```

Comment:
Can't leave a comment on them as they are in the original doc but the titles with gerunds ("ing" verbs used as nouns) - we should amend to imperatives (run, modify, generate, add).

Reply:
Also this title "Adding Unreal GDK configurations" needs to be:

"Adding GDK configurations" 

(We don't say "Unreal GDK" now.)